charge/clients/Client.py

```diff
@@ -1,21 +1,21 @@
 from typing import Type, Dict, Optional
 from abc import ABC, abstractmethod
-from charge.experiments.Experiment import Experiment
+from charge.tasks.Task import Task
 from charge._tags import is_verifier, is_hypothesis
 from charge.inspector import inspect_class
 import inspect
 import os
-from charge._to_mcp import experiment_to_mcp
+from charge._to_mcp import task_to_mcp
 import warnings
 import argparse
 import atexit
 import readline
 
 class Client:
     def __init__(
-        self, experiment_type: Experiment, path: str = ".", max_retries: int = 3
+        self, task_type: Task, path: str = ".", max_retries: int = 3
```

Comment:
```suggestion
        self, task: Task, path: str = ".", max_retries: int = 3
```

I think the `type` is unnecessary.